### ISSUE TITLE
Single Code for a CWE element ?

### ISSUE THREAD

**Issue by carolinerosin** [haffonist/igamt#2103]
We probably already discussed it, but I don't remember how this is supposed to work.
Should we be able to add "single code" to a CWE element ? Right now I can't do it (single code is greyed out at the field level, and I don't have access to "value set/single code" at my component level.

I am starting to think we should enable "value set/single code" for coded element datatypes (CE, CWE, CNE, CF), for components 1 (Identifier), 4 (Alternate identifier), 10 (Second Alternate identifier).

Here's my use case (that I can't implement with single code). I can only do "constant value".

![image](https://user-images.githubusercontent.com/13591511/29087554-54b80896-7c44-11e7-89f7-5e122ea6dfd0.png)

**Comment by Jungyubw** [comment]
@carolinerosin 

Current logic doesn't allow single code for complex datatype like CWE.

**Comment by carolinerosin** [comment]
Yes I know, but I was wondering if this is something we want to consider - maybe we can discuss with Rob during the next IGAMT meeting

**Comment by robnist** [comment]
We decided in IGAMT meeting to not allow this for now and to consider doing a pairwise single coded element binding for complex coded data type (e.g., CE). Pairwise means {CMD : HL70175}.

Can alreay do with CS but do we need a more powerful feature.
